### PR TITLE
Added Kotlin template, passes example test case

### PR DESCRIPTION
**Pull request by subash774**
New to Kotlin so please review code quality before merge.

### CODE REVIEW

File: Template.kt [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+    val testCases = sc.nextInt()
+
+    for(testCase in 1..testCases) {
+        val cmd = sc.next()
+        val n = sc.nextInt()
+        val a = DoubleArray(n)
+        for (i in 0 until n) {
+            a[i] = sc.nextDouble()
+        }
```
Comment (indy256) [inline Template.kt:21]: simplification: `val a = DoubleArray(n) { sc.nextDouble() }`

File: Template.kt [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                var sum = 0.0
+                for (v in a){
+                    sum += v
+                }
+                result = sum / n
+            }
+            else -> {
+                throw RuntimeException("Command not valid")
+            }
+        }
```
Comment (indy256) [inline Template.kt:41]: simplification:
```kotlin
        val result = when (cmd) {
            "median" -> a.sortedArray()[n / 2]
            "mean" -> a.sum() / n
            else -> throw RuntimeException("Command not valid")
        }
```

Comment (indy256) [inline Template.kt:41]:
```suggestion
        val result = when (cmd) {
            "median" -> a.sortedArray()[n / 2]
            "mean" -> a.sum() / n
            else -> throw RuntimeException("Command not valid")
        }
```

File: Template.kt [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+    val testCases = sc.nextInt()
+
+    for(testCase in 1..testCases) {
+        val cmd = sc.next()
+        val n = sc.nextInt()
+        val a = DoubleArray(n) { sc.nextDouble() }
+
+        val result : Double
+
```
Comment (indy256) [inline Template.kt:21]:
```suggestion
```

File: Template.kt [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+            "mean" -> a.sum() / n
+            else -> throw RuntimeException("Command not valid")
+        }
+        pw.printf("Case #$testCase: %.10f\n", result)
+        pw.flush()
+    }
+
+    pw.close()
+    sc.close()
+
```
Comment (indy256) [inline Template.kt:33]:
```suggestion
```

File: Template.kt [hunk 5]
```diff
@@ -0,0 +1,34 @@
+import java.io.*;
+import java.util.*;
+
+
+@Throws(Exception::class)
```
Comment (indy256) [inline Template.kt:5]:
```suggestion
```

File: Template.kt [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+fun main(args: Array<String>) {
+    val name = "A-small"
+    val path = ""
+
```
Comment (indy256) [inline Template.kt:9]:
```suggestion

    Locale.setDefault(Locale.US)

```

**Comment by subash774** [comment]
Thanks for the optimisation, should be good to go now.

**Review by indy256 — APPROVED** [review]
lgtm

**Pull request merged**